docs/stdlib/json.rst

```diff
@@ -166,10 +164,10 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
 .. eql:operator:: jsonidx: json [ int64 ] -> json
 
-    JSON array/string indexing.
+    Indexes an array or string of :eql:type:`json`.
 
-    The contents of JSON *arrays* and *strings* can also be
-    accessed via ``[]``:
+    This results in :eql:type:`json`. Contents of a JSON array or string can
```

Comment:
```suggestion
    The function returns the value at the given index as :eql:type:`json`.
```

I don't think the next sentence adds meaning beyond the suggested one-liner, so I'm removing it. Correct me if I'm wrong on that.